### DISCUSSION TITLE
Windows无法启动软件本体，会在任务栏里闪退一下然后消失

### DISCUSSION THREAD

**Discussion by Mivst** [nextai-translator/nextai-translator#1590] [Q&A]
昨天还能正常使用，今天无法打开软件，启动程序后会在后台闪一下然后结束运行，在日志里查不到报错，似乎和软件版本问题不大，尝试了之前的老版本，不知道是不是昨天清理C的缘故

**Comment by SuRu711** [comment]
我也是这样 不知道怎么解决 兼容性和防火墙都检查过了

**Comment by mjt1998** [comment]
2025年2月2号，我也遇到了该问题，请问你们找到解决办法了吗？
